components/HorizontalItem.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { View, StyleSheet, Text, TouchableWithoutFeedback } from 'react-native';
+import { colorGrey, colorImageBorder } from '../utils/colors';
+import FastImage from 'react-native-fast-image';
+import { IMAGE_BASE_URL } from '../network/NetworkData';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+const HorizontalItem = ({ id, imageUrl, title, description, onClick }) => {
+    return (
+        <TouchableWithoutFeedback onPress={() => onClick(id)}>
+            <View style={styles.container}>
+                <View style={styles.imageContainer}>
+                    <Icon
+                        name='image'
+                        color='#B5B5B5'
+                        size={44}
+                        style={[styles.image, {
+                            position: 'absolute',
+                            backgroundColor: '#DBDBDB',
+                            textAlign: 'center',
+                            textAlignVertical: 'center'
+                        }]} />
+                    <FastImage
+                        style={styles.image}
+                        source={{ uri: IMAGE_BASE_URL + imageUrl }} />
```

Comment:
Please check formatting. Seems to be off. 
Use prettier extension and make sure it's working properly.